changelogs/template.yml

```diff
@@ -0,0 +1,32 @@
+# Write your changelog entry here.  Every PR must have a changelog.
+#___DESCRIPTION___
```

Comment:
these lines can be removed https://github.com/massgov/mayflower/pull/812/files#diff-0de3b49620e59f1d729d4ca54219d53dR2-R4

Reply:
Removed this from the changelog template